mne/realtime/epochs.py

```diff
@@ -126,6 +126,14 @@ class RtEpochs(BaseEpochs):
         The events associated with the epochs currently in the queue.
     verbose : bool, str, int, or None
         See above.
+
+    Notes
+    -----
+    Calling `next()` on an `RtEpochs` object (as internally done when iterating
+    over the object) is blocking, i.e., waits for at most `isi_max` seconds
+    for a new epoch to be received.
+    Calling `get_data()` on an `RtEpochs` object immediately returns the epochs
```

Comment:
Did you want these to be separate paragraphs? They won't be unless you add an extra newline.

Maybe they should be bullets (created by adding `* ` or `- ` before start lines and indenting subsequent lines)?